src/lib/liquidity-curve.js

```diff
@@ -204,6 +204,7 @@ function intersectLineSegments (line0, line1) {
 
 class InvalidLiquidityCurveError extends Error {
   constructor (message) {
+    // message = message + ' points:' + this.points
```

Comment:
I forgot about this; it doesn't work as specified (something about this being referenced before super is called?).  Evan suggested that we print out all the points when related errors happen.  How would you accomplish that?